### ISSUE TITLE
Account suspended after downloading saved posts (400 Bad Request)

### ISSUE THREAD

**Issue by wi18b088** [instaloader/instaloader#1895]
TL;DR: If you are trying to download your saved photos from your private account, you might get suspended by Instagram.

Let me begin this short message by thanking the developers and contributors of instaloader for providing this clean, well-maintained tool that does what it is supposed to do. There is no comparable solution that offers this feature set with a high-level interface.

After developing a tool that helped me download my "saved" posts from Instagram myself a few years ago, I found instaloader as a working and easy solution to do the same in 2023. Unfortunately, times have changed - while batch-downloading content was pretty easy a few years ago, measures have become more strict. After downloading just ten posts from my saved album, I was hit with the dreaded 400 response code. My account was suspended, Instagram required me to provide a phone number and a "mugshot" holding a piece of paper with a number written on it so that they can generously consider unlocking my account again.

I'm pretty sure downloading content is against the terms of service, blocking users that attempt this is a reasonable approach by Meta. Firstly, I'd like to prevent others from having a similar experience. Secondly, I'd be curious if others were more successful or have working mitigation strategies against this.

Kind regards

**Comment by CatPeare** [comment]
I thank to the developer of this library as well, it was so useful back in the day. sadly similar thing happened to me. if anyone reads these messages please act with caution.

**Comment by blablable123456** [comment]
Hello, i got 2 acc banned because of the mugshot. I did them normally but after the second time they requested, they've just blocked me out of them. Some of those accounts had some good users i was looking after but now they are gone. If you got hit with something wait a week or so, its better of than just getting banned.

**Comment by madmoh91** [comment]
yeah same thing just happened to me, I just happened to be using another computer and used this as a quick solution cause I typically use 4k stockgram, not endorsing it as this is likely a change on instagrams part, so many years of not having to think about this.... lol
